lib/src/lints/avoid_returning_widgets/models/avoid_returning_widgets_parameters.dart

```diff
@@ -13,16 +13,11 @@ class AvoidReturningWidgetsParameters {
 
   /// Method for creating from json data
   factory AvoidReturningWidgetsParameters.fromJson(Map<String, dynamic> json) {
-    final exclude = <AvoidReturningWidgetsExclude>[];
-
-    final excludeList = json['exclude'] as Iterable? ?? [];
-    for (final item in excludeList) {
-      if (item is Map) {
-        exclude.add(AvoidReturningWidgetsExclude.fromJson(item));
-      }
-    }
     return AvoidReturningWidgetsParameters(
-      exclude: exclude,
+      exclude: ExcludedIdentifiersListParameter.fromJson(
+        excludeList:
+            json[ExcludedIdentifiersListParameter.excludeParameterName] as Iterable? ?? [],
+      ),
```

Comment:
1. We should probably make the `excludeList` parameter positional instead of named;
2. This piece repeats a lot -- let's create a default one that would parse from `dynamic` maybe?
```suggestion
      exclude: ExcludedIdentifiersListParameter.defaultFromJson(json),
```